ir/visitor.cpp

```diff
@@ -311,25 +429,23 @@ const IR::Node *Modifier::apply_visitor(const IR::Node *n, const char *name) {
     if (ctxt) ctxt->child_name = name;
     if (n) {
         PushContext local(ctxt, n);
-        if (visited->busy(n)) {
+        auto [busy, done] = visited->start(n, visitDagOnce);
```

Comment:
In the original code, `start` really was only used when starting processing on a node. Now it has the concept of "get status and start if not already started". Perhaps a name like `update` / `update_status` / `start_or_status`?

Reply:
It was mixed thing actually. There was "implicit" start in `Inspector` and slightly different thing in `Modifier`/ `Transform`. I thought about `try_start`. But if there are other options, then I'm all for it :)